### PR TITLE
Make sure preprocessor symbols are not overwritten when signing assemblies

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**

### CODE REVIEW

File: sonaranalyzer-dotnet/src/targets/CommonProperties.targets [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (pavel-mikula-sonarsource) [inline sonaranalyzer-dotnet/src/targets/CommonProperties.targets:1]: Can you put this content in `Directory.Build.props`? You can remove the explicit reference from the csproj files.

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
Pre-review

File: sonaranalyzer-dotnet/src/targets/CommonProperties.targets (hunk 1 above)
Comment (costin-zaharia-sonarsource) [reply sonaranalyzer-dotnet/src/targets/CommonProperties.targets]: Nice! This simplified the PR quite a bit.

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM, nice!

**Pull request merged**